frontend/packages/llama-stack-modular-ui/frontend/src/app/Chatbot/sourceUpload/ChatbotSourceUploadPanel.tsx

```diff
@@ -22,57 +20,53 @@ type ChatbotSourceUploadPanelProps = {
 };
 
 const ChatbotSourceUploadPanel: React.FC<ChatbotSourceUploadPanelProps> = ({
-  alert,
+  successAlert,
+  errorAlert,
   handleSourceDrop,
   selectedSource,
   selectedSourceSettings,
   removeUploadedSource,
   setSelectedSourceSettings,
 }) => (
-  <DrawerPanelContent isResizable defaultSize="400px" minSize="300px">
+  <>
     <AlertGroup hasAnimations isToast isLiveRegion>
-      {alert}
+      {successAlert}
+      {errorAlert}
     </AlertGroup>
-    <DrawerPanelBody>
-      <Title
-        headingLevel="h5"
-        style={{ fontWeight: 'bold', marginBottom: 'var(--pf-t--global--spacer--2xl)' }}
-      >
-        Sources
-      </Title>
-      <MultipleFileUpload
-        onFileDrop={handleSourceDrop}
-        dropzoneProps={{
-          accept: {
-            'application/msword': ['.doc'],
-            'application/pdf': ['.pdf'],
-          },
-          maxFiles: 1,
-        }}
-        aria-label="Source upload area"
-      >
-        <MultipleFileUploadMain
-          browseButtonText="Add"
-          titleIcon={<FileIcon />}
-          titleText="Drag and drop file here"
-          titleTextSeparator="or"
-          infoText="Upload a PDF or DOC file to be used in retrieval."
-        />
-        {selectedSourceSettings &&
-          selectedSource.map((file) => (
-            <MultipleFileUploadStatusItem
-              file={file}
-              key={file.name}
-              onClearClick={() => {
-                removeUploadedSource(file.name);
-                setSelectedSourceSettings(null);
-              }}
-              aria-label={`Uploaded file: ${file.name}`}
-            />
-          ))}
-      </MultipleFileUpload>
-    </DrawerPanelBody>
-  </DrawerPanelContent>
+    <MultipleFileUpload
+      onFileDrop={handleSourceDrop}
+      dropzoneProps={{
+        accept: {
+          'application/pdf': ['.pdf'],
+          'application/msword': ['.doc'],
+          'text/csv': ['.csv'],
+        },
+        maxFiles: 1,
+      }}
+      aria-label="Source upload area"
+    >
+      <MultipleFileUploadMain
+        browseButtonText="Upload"
+        titleIcon={<FileIcon />}
+        titleText="Drag and drop file here"
+        titleTextSeparator="or"
+        infoText="Accepted file types: PDF"
+        // infoText="Accepted file types: PDF, DOC, CSV"
+      />
+      {selectedSourceSettings &&
+        selectedSource.map((file) => (
+          <MultipleFileUploadStatusItem
```

Comment:
Maybe if we allow simultaneous multiple file upload this will have to be modified ?